reddit.js

```diff
@@ -1,5 +1,52 @@
 var reddit = require("./data.json")
 
+
 // Write your solutions below
 
-console.log(reddit["data"]);
\ No newline at end of file
+/*Part 1 - forEach - console.log all titles*/
+
+reddit.data.children.forEach(function (theTitle){
+	console.log(theTitle.data.title);
+})
+
+/*Part 2 - map - Return an array of permalinks,  to each so that they contain full http:// path to reddit*/
+var permalinks = [];
+
+reddit.data.children.forEach(function (child){
+	permalinks.push(child.data.permalink);
+});
+
+var permalinksHTTP = permalinks.map(function(http){
```

Comment:
Here, since you're iterating over each link in the ```permalinks``` array, I would recommend using ```link``` instead of ```http``` as your parameter, since the http is actually being prepended to each ```link```.